warehouse/redshift.go

```diff
@@ -46,6 +47,37 @@ func NewRedshift(c *config.Config) *Redshift {
 	}
 }
 
+func (rs *Redshift) getExportTableName(fullyQualifiedName bool) string {
```

Comment:
I wonder if it's better to encode the desire for an undecorated vs. a fully qualified table name into calling/not calling a function, rather than in a boolean parameter to this function. In other words, in all the places where you might call `getExportTableName(false)`, instead just directly refer to `rs.conf.Redshift.ExportTable`? You could then maybe rename this function to `qualifiedExportTableName()` (with no params). The intent of calling it (or not calling it) will be clear to the reader of the calling code without having to read this function. It will also simplify the below if statement a bit.

Same for `getSyncTableName` below.

Reply:
Great point - I'll fix this up.